liquigraph-examples/dagger2/src/test/java/org/liquigraph/examples/dagger2/ApplicationTest.java

```diff
@@ -53,5 +51,25 @@ private static int availablePort() {
         }
     }
 
+    private static Neo4jRule withVersionAwareConfig(Neo4jRule neo4jRule) {
+        if (isVersionInRange("3.0", "3.1")) {
+            return neo4jRule
+                .withConfig("dbms.connector.0.enabled", "false") /* BOLT */
+                .withConfig("dbms.connector.1.address", "localhost:" + availablePort()); /* HTTP */
+        }
+        return neo4jRule
+            .withConfig("dbms.connector.bolt.enabled", "false")
+            .withConfig("dbms.connector.http.enabled", "true")
+            .withConfig("dbms.connector.http.address", "localhost:" + availablePort());
+    }
 
+    private static boolean isVersionInRange(String... versionPrefix) {
```

Comment:
May I suggest `isVersionIn()` instead of `isVersionInRange()` ? "Range" suggests lower and upper bounds (as does the way it's called with `isVersionInRange("3.0", "3.1")`), whereas you actually need to pass all the versions that should match.